tests/MongoDB.Driver.Tests/Specifications/crud/CrudOperationTestBase.cs

```diff
@@ -75,7 +75,7 @@ protected virtual void AssertOutcome(BsonDocument outcome, IMongoDatabase databa
 
         protected abstract bool TrySetArgument(string name, BsonValue value);
 
-        protected abstract void Execute(IMongoCollection<BsonDocument> collection, BsonDocument outcome, bool async);
+        protected abstract void Execute(IMongoDatabase database, IMongoCollection<BsonDocument> collection, BsonDocument outcome, bool async);
```

Comment:
The change in signature is because some operations need just a database. For those operations collection will be null.